server/channels/jobs/cleanup_desktop_tokens/worker.go

```diff
@@ -9,28 +9,19 @@ import (
 	"github.com/mattermost/mattermost/server/public/model"
 	"github.com/mattermost/mattermost/server/public/shared/mlog"
 	"github.com/mattermost/mattermost/server/v8/channels/jobs"
-	"github.com/mattermost/mattermost/server/v8/channels/store"
-	"github.com/mattermost/mattermost/server/v8/platform/services/configservice"
 )
 
 const jobName = "CleanupDesktopTokens"
 const maxAge = 5 * time.Minute
 
-type AppIface interface {
-	configservice.ConfigService
-	ListDirectory(path string) ([]string, *model.AppError)
-	FileModTime(path string) (time.Time, *model.AppError)
-	RemoveFile(path string) *model.AppError
-}
-
-func MakeWorker(jobServer *jobs.JobServer, store store.Store) *jobs.SimpleWorker {
+func MakeWorker(jobServer *jobs.JobServer) *jobs.SimpleWorker {
 	isEnabled := func(cfg *model.Config) bool {
 		return true
 	}
 	execute := func(logger mlog.LoggerIFace, job *model.Job) error {
 		defer jobServer.HandleJobPanic(logger, job)
 
-		return store.DesktopTokens().DeleteOlderThan(time.Now().Add(-maxAge).Unix())
+		return jobServer.Store.DesktopTokens().DeleteOlderThan(time.Now().Add(-maxAge).Unix())
```

Comment:
I found some weirdness with the last job I created so I decided to fix that here :P

Reply:
@devinbinnie is this for the desktop auth? Should we include a smoke test for this in the QA steps just to make sure there is no regressions here?

Reply:
@marianunez We can, but this is just a name change. Don't really think it's needed, but 0/5

Reply:
Let's just have at least one manual pass to make sure everything is still working as expected

Reply:
@yasserfaraazkhan can you do a quick pass?

Reply:
cc/ @yasserfaraazkhan ^